sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProto.java

```diff
@@ -746,6 +755,12 @@ private static void fieldDescriptorFromTableField(
               return ChronoUnit.MICROS.between(
                   Instant.EPOCH, Instant.ofEpochMilli(Long.parseLong((String) value)));
             } catch (NumberFormatException e2) {
+              if (((String) value).endsWith("UTC")) {
```

Comment:
we should not catch an exception to determine we should use an alternate date time formatter

Reply:
that was already the current implementation, will change to have one date time formatter to parse from.